partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/operator/OpArithmeticTest.kt

```diff
@@ -51,4 +55,89 @@ class OpArithmeticTest : PartiQLTyperTestBase() {
 
         return super.testGenPType("arithmetic", tests, argsMap)
     }
+
+    @TestFactory
+    fun plus(): Stream<DynamicContainer> {
```

Comment:
Self-review: I copied the `arithmetic` test from above to split out the plus/minus planner tests, since we can now add intervals to datetimes and vice-versa.